### PR TITLE
if: add if conditions with examples

### PR DESCRIPTION
**Pull request by a-b**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/if.md [hunk 1]
```diff
@@ -9,3 +9,19 @@
 - Full if syntax:
 
 `if {{condition}}; then echo "true"; else echo "false"; fi`
+
+- List available if conditions:
+
+`help test`
+
+- If given variable is empty:
```
Comment (sbrl) [inline pages/common/if.md]: Perhaps this will read better?
```suggestion
- Test if a given variable is empty:
```

File: pages/common/if.md [hunk 2]
```diff
@@ -9,3 +9,19 @@
...
+
+`if [[ -z $GIT_BRANCH ]]; then echo "true"; else echo "false"; fi`
+
+- If file exists:
```
Comment (sbrl) [inline pages/common/if.md]: Perhaps this will read better?
```suggestion
- Test if a file exists:
```

File: pages/common/if.md [hunk 3]
```diff
@@ -9,3 +9,19 @@
...
+
+`if [[ -e {{filename}} ]]; then echo "true"; else echo "false"; fi`
+
+- If directory not exists:
```
Comment (sbrl) [inline pages/common/if.md]: Perhaps this will read better?
```suggestion
- Test if a directory does not exist:
```

**Comment by a-b** [comment]
Thanks for your input @sbrl ! @pxgamer @schneiderl - should be good to merge.

**Pull request merged**